var/spack/repos/builtin/packages/hip-rocclr/package.py

```diff
@@ -92,6 +94,14 @@ def url_for_version(self, version):
              placement='opencl-on-vdi',
              when='@4.0.0')
 
+    resource(name='opencl-on-vdi',
```

Comment:
Any news on moving this to a separate package? I'd like to use libOpenGL.so too :upside_down_face:

Reply:
i have started on this package and i see a couple of build errors which require some fixes. I will fix them and upstream it soon.